lib/generateConfig.js

```diff
@@ -26,7 +26,7 @@ async function generateConfig({ envConfig, envFile }) {
 
   const content = await readFileP(envFile, 'utf8');
 
-  content.split('\n').forEach((line) => {
+  content.split('\n').map(l => l.split('#')[0].trim()).forEach((line) => {
```

Comment:
Can you split these into multiple lines to make them more readable?

Something like:
```
content.split('\n')
   .map()
   .forEach()
```